avocado_vt/options.py

```diff
@@ -466,6 +466,21 @@ def _process_general_options(self):
         self._process_bridge_mode()
         self._process_only_type_specific()
 
+    def _process_backend_specific_options(self, vt_type):
+        """
+        Calls for processing of backend specific options
+        """
```

Comment:
Could this docstring use one-liner comment like `"""Calls for ..."""`? I find it more readable.